src/VecTargets.jl

```diff
@@ -1,9 +1,15 @@
 module VecTargets
 
-include("bnormals.jl")
 using BSON, Parameters, Distributions, DistributionsAD
-using Random: GLOBAL_RNG, shuffle
+import Distributions: Distributions, VariateForm, ValueSupport, Discrete, Continuous, Distribution, ContinuousMultivariateDistribution, logpdf, pdf, cdf, invlogcdf, ccdf, rand, mean, std, var, mode, minimum, maximum
```

Comment:
That's a veeeeeery long line. Also, might it make sense to use `import` only for the ones that are being extended with new methods?